### ISSUE TITLE
Cannot use this repository

### ISSUE THREAD

**Issue by alexandrudima** [Microsoft/vscode-docs#2525]
This is the 4th time I try to clone this repo.

* I am running:
  * `GIT_LFS_SKIP_SMUDGE=1 git clone https://github.com/Microsoft/vscode-docs.git`
  * `cd vscode-docs`
  * `git lfs install`
  * `git checkout vnext`
  * `git lfs pull -I "release-notes/images/1_3*/*"`

And then, when I do `git status` I have one modified file, which prohibits me from checking in my release notes... I cannot undo this modified file, regardless of what I do.

```
Alexs-MBP:src alex$ GIT_LFS_SKIP_SMUDGE=1 git clone https://github.com/Microsoft/vscode-docs.git
Cloning into 'vscode-docs'...
remote: Enumerating objects: 45, done.
remote: Counting objects: 100% (45/45), done.
remote: Compressing objects: 100% (32/32), done.
remote: Total 3018 (delta 26), reused 29 (delta 13), pack-reused 2973
Receiving objects: 100% (3018/3018), 2.25 MiB | 1.16 MiB/s, done.
Resolving deltas: 100% (327/327), done.
Alexs-MBP:src alex$ cd vscode-docs
Alexs-MBP:vscode-docs alex$ git lfs install
Updated git hooks.
Git LFS initialized.
Alexs-MBP:vscode-docs alex$ git checkout vnext
Downloading docs/languages/images/java/java-debug.mp4 (468 KB)
Downloading docs/languages/images/java/tomcat.mp4 (1.4 MB)
Downloading release-notes/images/1_33/extension-dep-install.png (64 KB)
Branch vnext set up to track remote branch vnext from origin.
Switched to a new branch 'vnext'
Encountered 1 file(s) that should have been pointers, but weren't:
	release-notes/images/1_33/sub-sessions.png
Alexs-MBP:vscode-docs alex$ git lfs pull -I "release-notes/images/1_3*/*"
Alexs-MBP:vscode-docs alex$ % (63/63), 58 MB | 3.8 MB/s
Alexs-MBP:vscode-docs alex$ git status
On branch vnext
Your branch is up-to-date with 'origin/vnext'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	modified:   release-notes/images/1_33/sub-sessions.png

no changes added to commit (use "git add" and/or "git commit -a")
```

**Comment by isidorn** [comment]
I have pushed the sub-sessions.png
I think I have installed the git lfs properly following these steps https://github.com/Microsoft/vscode-docs/issues/2524#issuecomment-477945792

**Comment by alexandrudima** [comment]
I see, I was supposed to commit and push that permanent change. Perhaps the README can explain this stuff or perhaps there can be a pre commit hook to prevent such bad states from happening.

**Comment by alexandrudima** [comment]
Fixed via 1488344b074ad0deb3fb8abac33a3cd2926f9a61
